tock/tock/templates/utilization/utilization_analytics.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% block navigation %}
+  {% include "_navigation.html" %}
+{% endblock %}
+
+{% block content %}
+  <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
```

Comment:
Let's bring this in house and serve ourselves for consistency by pulling the version we need into `tock/tock/static/vendor`

https://github.com/18F/tock/blob/main/docs/dependency-management.md#production